HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -237,6 +237,16 @@ def customizeHLTfor42497(process):
     return process
 
 
+def customizeHLTfor42817and42860(process):
```

Comment:
```suggestion
def customizeHLTfor42869(process):
```

this PR name has to be used here.

Reply:
How can one put in the PR number here before making the PR?